website_blog_group_old_archives/controllers/website_blog.py

```diff
@@ -70,7 +70,7 @@ def blog(self, blog=None, tag=None, page=1, **opt):
         self._date_begin, self._date_end = opt.get('date_begin'), opt.get('date_end')
         self._tag = tag
         result = super(website_blog, self).blog(
-                blog=blog, tag=tag, page=page, **opt)
+                blog=blog, tag=tag, page=page, **opt.get('opt'))
```

Comment:
the current opt is like opt={opt{start_date:'1-1-2017','end_date:'1-2-2017'},}
 how did we end up with this nested dict? is there a possibility of cleaning up before, when the function is called?